index.html

```diff
@@ -182,19 +187,27 @@ <h3>Skills</h3>
                 </ol>
               </li>
             </ul>
-          </div> </div>
+          </div> </div></div>
           <div class="container4">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <h2 id="t">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
           </div>
           <div class="container5">
             <form action="https://formspree.io/f/myyaolbw" method="POST">
```

Comment:
Great job on implementing the previous review requests! It's evident that you've made improvements based on the feedback. However, there are still a few adjustments that need to be made to ensure a better user experience.

1. Responsive Design: While the section looks good on a 1024px device, it's crucial to make it responsive across different screen sizes. The mobile version should be optimized from 375px to 768px screen size. The desktop version should be optimized from 768px to at least 1440px, considering the Figma design's width. You can even extend the responsiveness beyond 1440px if desired. Please refer to the video provided to see how the form does not look good on different desktop screen sizes. Make the necessary adjustments to your code to achieve responsiveness and ensure the design looks great across various devices.

2. Vertical Scroll: Currently, there is a vertical scroll present on all screen sizes. It's important to remove the vertical scroll as it is not part of the design and can negatively impact the user experience. Please review your code and make the necessary adjustments to eliminate the vertical scroll.

[Tsige-s-Personal-Portifolio.webm](https://github.com/tsigewossne/portfolio/assets/55840999/ce4317fc-0dc4-4ccc-90d9-aa2b506de6ca)


Keep up the great work and remember to prioritize responsiveness and a seamless user experience. Feel free to reach out if you have any questions or need further assistance! 👍